doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
Oh sorry. After writing out a description (as best as I could understand it), I'm not sure whether it will help clarify the docs, but maybe I'm just thinking along the wrong lines.

Let's just say there are two ticks, which can be written as `abcd` and `efgh` (where each of those letters is a single 0-9 digit). Then pick some other value between them `jk00` so that the ticks are `abcd - jk00 => -nopq` and `efgh - jk00 => rstu`. If you can pick `jk00` such that `n == o == r == s == 0`, then the offset is used.

The new setting means the "other value" must be `jklm` and additional conditions of `p == q == t == u == 0` must also be satisfied (which it wouldn't be possible in this case, thus fixing dates.)